src/main/python/janusgraph_python/core/attribute/GeoPredicate/Geo.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2018 JanusGraph Python Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from .GeoContains import GeoContains
+from .GeoWithin import GeoWithin
+from ...datatypes.GeoShape import GeoShape
+
+
+class Geo(object):
+    """
+    This class defines all the Geo Predicates.
+    """
+
+    contains = GeoContains()
+    within = GeoWithin()
+
+    @staticmethod
+    def geoContains(value):
+        """ The class is used for JanusGraph geoContains predicate.
+
+        GeoContains predicate holds true when one object, is contained by another. Considering the example in docs,
+        the following statement holds true : India (GeoShape) contains (GeoContains) New Delhi (GeoShape)
+
+        Args:
+            value (GeoShape): The GeoShape to query for and return all results which are present inside this GeoShape
+
+        Returns:
+            Any
```

Comment:
`Any`? Shouldn't this include the return type (`bool`) and tell the reader when it returns true and when false?

Reply:
@FlorianHockmann : I think it will not be `Any`, but might be of type `GeoShape` I guess. Because, if the query returns to True, the result of Predicate is actual Geometrical object right? Either a single of list depending on how it is used with `next()` or `toList()`

Reply:
Basically the same as for `geoContains`: Predicates are functions that only return a boolean and Gremlin steps use these predicates to filter the objects they are currently traversing. The TinkerPop reference docs [contain a more detailed description](http://tinkerpop.apache.org/docs/current/reference/#a-note-on-predicates).

Reply:
Ah. Thanks for the clarification. What I've done now is I've updated the return type for both `Geo` and `Text` predicate classes accordingly. 
But, if that is the case, shouldn't it be same in your PR? I see you have specified return type of Text Predicates as just `Text Predicate` [here](https://github.com/JanusGraph/janusgraph-dotnet/blob/6918516f78b692ee608b558bf792446c4831f9b3/src/JanusGraph.Net/Text.cs#L41)